src/java_bytecode/CMakeLists.txt

```diff
@@ -15,8 +15,7 @@ add_custom_command(TARGET core_models_files
                    PRE_BUILD
                    COMMAND java-converter ${CORE_MODELS_JAR} "java_core_models" ${CMAKE_CURRENT_SOURCE_DIR})
 
-file(GLOB_RECURSE c_sources "*.c")
-file(GLOB_RECURSE sources "*.cpp")
+file(GLOB sources "*.cpp")
```

Comment:
Commit can be squashed into https://github.com/cesaro/cbmc/pull/5/commits/068c730d5164434a1b3c5894d57b04e361cef962